test/pasta-curves-tests.t.sol

```diff
@@ -78,30 +78,93 @@ contract PastaCurvesContractTests is Test {
         assertEq(bytes32(yExpected), bytes32(msm.y));
     }
 
-    // TODO: add test for pow small with random exponent
+    function testVestaMSM() public {
+        /*
```

Comment:
Don't you want to add Rust snippets, in order to understand how hardcoded values were generated? Obviously, that's not mandatory as they can be copied from Pallas tests with minimal changes. Ideally, we would like to have this test vectors evaluation using auto-generated values from some Rust application - it would help us to preserve compatibility issues at early stages, when for example the version of `pasta_curves` Rust crate changes (which we treat as reference). But for now, having snippets is useful, I think

Reply:
All of the Vesta tests were exactly copies of the Pallas tests with all the reference to Pallas replaced with Vesta. I semi-jokingly-semi-seriously added the `sed` command to make those changes.

I can add the code snippets in, but I'm not sure it's very helpful when they exactly mirror the Pallas tests. I did some daydreaming last week about how we could generate and import the test vectors automatically from a Rust project, but it seemed like it would be a nontrivial undertaking and instead got back to writing tests.

Maybe as a temporary solution, we could add the Rust code that generates the test vectors as a git submodule in this repo?